content/docs/using-github-directly/index.mdx

```diff
@@ -0,0 +1,303 @@
+
+When you have your content in the index.mdx file, save it (although we also recommend saving this iteratively in your GitHub editor as a matter of course), and if you are running a local version of DARIAH-Campus in your browser you should see your event appear in the ‘Resources’ page, as in here:
+
+![Copying the resource template folder](images/DC_PFndr_img4_inBrowser.png)
+
+Once you are happy with your draft hosted resource, you can submit it to the DARIAH-Campus team via GitHub.
+
+### Event
+
+Once you have set up DARIAH-Campus on your computer, you can use the `event-template-folder` as a basis for your contribution.
+
+You can find this template folder in your local working copy in the ‘templates’ folder i.e.
+
+`dariah-campus > templates > events > event-template-folder`
+
+![Copying the event template folder](images/DC_CapEvent_img1_copy.png)
+
+**Copy** (do not ‘drag and drop’) the entire folder, and then **paste** it into the ‘content/events’ folder in your fork.
+
+![Copying the resource template folder](images/DC_CapEvent_img2_paste.png)
+
+When you have these templates in your local ‘events’ fork, you can start editing. You can see an example of the edited `index.mdx` file here:
+
+![Copying the resource template folder](images/DC_CapEvent_img3_ingreen.png)
+
+Within this file, you will write all the content you want to include in your captured event. If you need a little more assistance in how to format this, or to include certain elements, such as videos, images, or tables, please take a look at our ‘Writing in Markdown’ guide in the ‘GitHub Workflow section.
+
+When you have your content in the index.mdx file, save it (although we also recommend saving this iteratively in your GitHub editor as a matter of course), and if you are running a local version of DARIAH-Campus in your browser you should see your event appear in the ‘Events’ page, as in here:
+
+![Copying the event template folder](images/DC_CapEvent_img4_inBrowser.png)
+
+Once you are happy with your 'Captured Event', you can ‘commit’ it via GitHub to the DARIAH-Campus team.
+
+## Creating content
+
+### Setting the Metadata for Citation
+
+The metadata of your resource is important as it forms the basis of the **citation** recommendation that you will see at the top of all the training resources. In order to ensure that this citation is correct, you will need to make sure you have the following information to hand:
+
+- The Title of Your Resource
+- The names of any authors, contributing authors or editors of your resource
+- The 'type' of training resource it is, e.g. 'training module', 'video'
+- The URL for where the resource is currently located. This is called the '**remoteUrl**' in the metadata (this only applies for External Resources)
+- The date your training resource was originally published. This is called the '**remotePublicationDate**' in the metadata (this only applies for External Resources)
```

Comment:
```suggestion
- The date your training resource was originally published. This is called the '**remote.date**' in the metadata (this only applies for External Resources)
```

Reply:
did I get this wrong or is this a change in the system?

Reply:
this is a small change in our schema, so all `remote` fields are grouped under one object